src/components/BlogCard/index.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../style/variables";
+
+.blogPost {
+  height: 20rem;
+  border-radius: 1rem !important;
```

Comment:
should be the variable that we use for border radius

Reply:
if that not suitable, I suggest to create one for image and use it

Reply:
it looks really small and not like the other cards, the border-radius depends on the size of the element :


![Screen Shot 2020-11-22 at 12 25 01 PM](https://user-images.githubusercontent.com/58190902/99899922-c629c780-2cbd-11eb-8170-abe035e56648.png)

![Screen Shot 2020-11-22 at 12 25 07 PM](https://user-images.githubusercontent.com/58190902/99899934-d5a91080-2cbd-11eb-8e6b-b09e93cbe8b3.png)

![Screen Shot 2020-11-22 at 12 25 04 PM](https://user-images.githubusercontent.com/58190902/99899933-d3df4d00-2cbd-11eb-8d4f-0673ee952fe1.png)

Reply:
the first image I used radius in the variables

Reply:
you can create one for all cards and images, so we can control it in one place